mods/ra/maps/allies-05a/map.yaml

```diff
@@ -1592,7 +1592,7 @@ Rules:
 		-SpawnMPUnits:
 		-MPStartLocations:
 		LuaScript:
-			Scripts: allies05a.lua, AI.lua
+			Scripts: allies05a.lua, AI05a.lua
```

Comment:
and likewise for all the other files that you refer to only by their mission number.